src/ui/application-window.vala

```diff
@@ -251,6 +254,23 @@ namespace Peek.Ui {
       record_button.set_label (_("Record as %s").printf (format_name));
       pop_format.hide ();
     }
+    
+    private void update_time () {
+      int seconds=0;
+      Timeout.add_seconds_full (GLib.Priority.LOW,1, () => {
+        seconds +=1;
+        if (is_recording && stop_button.sensitive == true) {
```

Comment:
I am just curious, why is stop_button.sensitive needed to be checked?

Reply:
To be honest It is not needed, I was doing some testing and left it in there. It will be removed.